target_postgres/target.py

```diff
@@ -329,7 +330,9 @@ def _process_record_message(self, message_dict: dict) -> None:
         """
         stream_name = message_dict["stream"]
         if self.mapper.stream_maps.get(stream_name) is None:
-            raise Exception(f"Schema message has not been sent for {stream_name}")
+            raise RecordsWithoutSchemaException(
```

Comment:
https://github.com/meltano/sdk/blob/main/singer_sdk/target_base.py#L325 

I think we can dump this whole function, and even the test